### ISSUE TITLE
Per project settings

### ISSUE THREAD

**Issue by davisford** [fbzhong/sublime-jslint#18]
It would be great to have per-project jslint settings, similar to how [Sublime Clang plugin works](https://github.com/quarnster/SublimeClang/blob/master/SublimeClang.sublime-settings#L151)

**Comment by ghost** [comment]
Yes, a `sublime-jslint.sublime-settings` file (or some other name) would be most useful. Will look into this.
